pom.xml

```diff
@@ -28,6 +28,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <skipTests>true</skipTests>
```

Comment:
Sorry, I have updated my own fork without thinking about this open P/R. I'll fix that.